lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -793,3 +807,37 @@ PYI_POST_OFFICE:
     end:
       targetJourney: INELIGIBLE
       targetState: INELIGIBLE
+
+# Mitigation journey (03) same session - invalid-dl
+MITIGATION_03_OPTIONS:
+  response:
+    type: page
+    pageId: pyi-driving-license-no-match-another-way
+    mitigationStart: invalid-dl
+  events:
+    next:
+      targetState: CRI_UK_PASSPORT_J2
+    end:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE
+
+# Mitigation journey (04) separate session - invalid-dl
+MITIGATION_04:
+  response:
+    type: page
+    pageId: pyi-driving-license-no-match
```

Comment:
I think that `mitigationStart: invalid-dl` should be defined on this event, and not the `MITIGATION_04_IDENTITY_START_PAGE` state.

Reply:
thought this an intermediate page .......actual option to select Passport journey is in next page pyi-continue-with-passport...thats way added in state MITIGATION_04_IDENTITY_START_PAGE

Reply:
Yep fair, looking at the content on those pages I think you're right.